libs/patchwheel/parse.go

```diff
@@ -39,6 +40,7 @@ func calculateNewVersion(info WheelInfo, mtime time.Time) (newVersion, newFilena
 // The function does not try hard to validate if the format is correct, it tries to parse whatever is available.
 // However, it does require 5 or 6 components in the filename.
 func ParseWheelFilename(filename string) (WheelInfo, error) {
+	filename = path.Base(filename)
```

Comment:
Shouldn't it be filepath.Base to be OS specific?

Reply:
You need to use `filepath.Base` on Windows unless you know the path is slash-separated.

Reply:
good point, thanks https://github.com/databricks/cli/pull/2521/commits/f408143a13a986b060bd2e4d41d2bd7fda5c57a9